docs/components/switch/index.html

```diff
@@ -0,0 +1,98 @@
+---
+title: Switch
+minver: 0.22.0
+also:
+components/switch/test.html: Testing - Switch
+elements/hx-switch: <hx-switch>
+  ---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+  {# TODO: add component description #}
+  {% endblock %}
+
+  {% block content %}
+  <section>
+    <header>
+      <h2 id="basic-checkbox">Switch</h2>
+      <p>A switch is used when a user needs to make a binary decision, such as turning a setting on or off or setting a preference enabling or disabling a device or system</p>
+    </header>
+
+    <div class="example" id="vue-switchDemo" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Options</legend>
+
+            <hx-checkbox-control>
+              <input id="chkIsDisabled" type="checkbox" v-model="isDisabled" />
+              <label for="chkIsDisabled">
+                <hx-checkbox></hx-checkbox>
+                Disabled
+              </label>
+            </hx-checkbox-control>
+          </fieldset>
+        </form>
+      </header>
+
+      <div>
+        <hx-switch-control class="switch">
+          <input type="checkbox" id="switchTest" :disabled="isDisabled" />
+          <label for="switchTest">
+            <hx-switch onlabel="on" offlabel="off" aria-labelledby="switchTest" class="slider round">
```

Comment:
Do we still need these classes (try removing):

```js
class="slider round"
```

You should be able to modify Switch component styles, so that these are not required.